Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -101,12 +184,57 @@ public final class ProjectPamphletViewController: UIViewController {
     self.viewModel.inputs.willTransition(toNewCollection: newCollection)
   }
 
-  private func setInitial(constraints: [NSLayoutConstraint?], constant: CGFloat) {
+  // MARK: - Private View Setup Functions
+  private func configureSublayers() {
+    if let shapeLayer = self.backThisProjectContainerView.layer.sublayers?.first as? CAShapeLayer {
```

Comment:
Little bit confused by what's going on here? If we're reaching in for this layer shouldn't we just keen a reference to it? Also, if it's the first layer then we could probably just check if the sublayer count is greater than 0 🤔

Reply:
The `backthisProjectContainerView`'s layer starts off with one sub-layer: the button that's added as a subview. Once we add the shape layer, the count goes up to 2. Because of this, our options are either check if `backThisProjectContainerView.layer.sublayers.count == 1` or check that the first sublayer is the shape layer we added. Imo checking that the first sublayer is a `CAShapeLayer` is more specific, which is why I decided to go with that.

Reply:
@justinswart take a look at the updated code, now we're just storing a reference to the layer as you suggested and updating the layer's `path`